code/modules/mob/living/simple_animal/abnormality/aleph/melting_love.dm

```diff
@@ -123,21 +117,24 @@
 /mob/living/simple_animal/hostile/abnormality/melting_love/proc/SlimeConvert(mob/living/carbon/human/H)
 	if(!istype(H))
 		return FALSE
+	if(H.has_status_effect(STATUS_EFFECT_MELTYLOVE))
+		//The status effect should explode them eventually. If not we have a bigger problem.
+		return FALSE
 	visible_message(span_danger("[src] glomps on \the [H] as another slime pawn appears!"))
 	new /mob/living/simple_animal/hostile/slime(get_turf(H))
-	H.gib()
+	H.gib(FALSE, TRUE, TRUE)
 	return TRUE
 
 /* Qliphoth things */
 /mob/living/simple_animal/hostile/abnormality/melting_love/SuccessEffect(mob/living/carbon/human/user, work_type, pe)
 	. = ..()
-	if(prob(33) && user == gifted_human)
+	if(prob(33) && user.has_status_effect(STATUS_EFFECT_MELTYLOVE))
 		datum_reference.qliphoth_change(1)
 	return
 
 /mob/living/simple_animal/hostile/abnormality/melting_love/NeutralEffect(mob/living/carbon/human/user, work_type, pe)
 	. = ..()
-	if(prob(50))
+	if(prob(50) && user.has_status_effect(STATUS_EFFECT_MELTYLOVE))
```

Comment:
change not documented in the PR's changelog, nor in the records of melting love

Reply:
Oh i thought it needed the user effect his is a accident.